etc/omero.properties

```diff
@@ -662,56 +724,58 @@ omero.ldap.group_filter=(objectClass=groupOfNames)
 
 omero.ldap.group_mapping=name=cn
 
-## Without a prefix the "new_user_group" property
-## specifies the name of a single group which all
-## new users will be added to.
+# Without a prefix the "new_user_group" property
+# specifies the name of a single group which all
+# new users will be added to.
+#
+# Other new_user_group strings are prefixed with
+# ":x: and specify various lookups which should
+# take place to find one or more target groups
+# for the new user.
+#
+# :ou: uses the final organizational unit of a user's dn
```

Comment:
This needs to be enclosed using backquotes cf https://github.com/openmicroscopy/ome-documentation/blob/develop/omero/sysadmins/server-ldap.txt#L124